### PR TITLE
 Add sample scenarios for testing the operator with README

### PR DESCRIPTION
**Pull request by RameshaKaru**
This PR adds sample resource artifacts to test scenarios. Following functions can be tested.
Adding API
Update API
Delete API
Analytics
OAuth2 flow
Ratelimiting
Security kind

Instructions to follow are included in the README.

Further, this PR adds the show command to view API details and related swagger

### CODE REVIEW

File: apim-operator/deploy/scenarios/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - ***kubectl apply -f oauth_security.yaml***
+    - Note: User must deploy two secrets with the certificates. The following command can be used to a create secret from the certificate pem file.
+    - ***kubectl create secret generic wso2am260-secret --from-file=wso2am260.pem -n=wso2-test-ns1***
+    - ***kubectl create secret generic wso2am260-secret --from-file=wso2am260.pem -n=wso2-test-ns2***
+
+- Do hostmapping for APIM and analytics servers.  <br />
+    - ***kubectl apply -f host_mapping.yaml***
+
+- Deploy two APIs in the created namespaces.  <br />
+    - ***kubectl add api myapi1 --from-file=swagger_ns1_oauth.yaml -n=wso2-test-ns1***
```
Comment (pubudu538) [inline apim-operator/deploy/scenarios/README.md]: We should not use wso2-system namespace to create any API.

Comment (pubudu538) [reply apim-operator/deploy/scenarios/README.md]: Or any kind.

File: apim-operator/deploy/scenarios/scenario1/apim_analytics_configs.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  port: "9444"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apim-config
+  namespace: wso2-system
+data:
+  verifyHostname: "true"
```
Comment (pubudu538) [inline apim-operator/deploy/scenarios/scenario1/apim_analytics_configs.yaml]: These artifacts are part of the deployment and used by devOps people to setup the environment.

File: apim-operator/deploy/scenarios/README.md (hunk 1 above)
Comment (RameshaKaru) [reply apim-operator/deploy/scenarios/README.md]: wso2-system namespace was not used for api creation, but the test namespaces created in the 1st step(wso2-test-ns1 and wso2-test-ns2) were used.

File: apim-operator/deploy/scenarios/scenario1/apim_analytics_configs.yaml (hunk 2 above)
Comment (RameshaKaru) [reply apim-operator/deploy/scenarios/scenario1/apim_analytics_configs.yaml]: Removed the artifact now. (This was added to demonstrate the analytics enabled scenario with a shorter usage uploading cycle)

**Pull request merged**